oauth2client/client.py

```diff
@@ -279,8 +280,7 @@ def new_from_json(cls, s):
       An instance of the subclass of Credentials that was serialized with
       to_json().
```

Comment:
In that case, let's be explicit about supporting both? Documentation that just says "string" suggests to the Python 3 reader that we're not describing the value completely.